app/assets/javascripts/lesson_planner/lesson_planner.jsx

```diff
@@ -188,6 +188,19 @@ EC.LessonPlanner = React.createClass({
     });
   },
 
+	// TODO: remove staging from the URL or build a regex that grabs everythig before the third slash using window.location.href
+	getInviteStudentsUrl: function() {
+		return ('/teachers/classrooms/' + $(".tab-pane").data().classroomId + '/invite_students');
+	},
+
+	studentsPresent: function() {
+		return $(".tab-pane").data().students;
```

Comment:
@RyanNovas for readability I like to use the convention of 'if its a boolean, the name should indicate that' so perhaps `students` should be `areThereStudents` or `studentsPresent`